app/grandchallenge/reader_studies/views.py

```diff
@@ -214,8 +216,37 @@ def get_context_data(self, **kwarsg):
         return context
 
 
+class QuestionOptionMixin(object):
+    def validate_options(self, form, _super):
+        context = self.get_context_data()
+        options = context["options"]
+        if form.data["answer_type"] in ["CHOI", "MCHO"] and not any(
```

Comment:
Use `ANSWER_TYPE_CHOICE` and `ANSWER_TYPE_MULTIPLE_CHOICE` instead of referring to the keys directly

Reply:
Ah, yes, forgot to clean this part up, sorry!